.github/workflows/Tests.yaml

```diff
@@ -0,0 +1,222 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Continuous integration tests that pull requests are required to pass
+name: Tests
+
+on:
+  # We run all jobs on `pull_requests` events.
+  pull_request:
```

Comment:
I'm confused, the documentation you linked reads like requesting a reviewer will cause the PR to be rebuilt and have tests rerun on it. Is that really what happens?

Reply:
No, please see comment above: https://github.com/sxs-collaboration/spectre/pull/1804#discussion_r348638474

Reply:
Right, but your comment on line 8 I think could be clarified. How about:
`# We run all jobs when pull requests are opened, commits are pushed, or re-opened after being closed.`

I'd also like a comment before the `on` along the lines of:
```
# Note that by default only PRs are tested and the develop branch of the main repository.
# The tests can be run on forks if actions are set up there.
```

This raises the question: since `push` is set only to develop branches, does this mean that only the develop branch would be tested on forks?

Reply:
Currently I restrict it to `develop`, yes. But since actions only run on forks when explicitly enabled, I suppose it’s better to remove the restriction. That means the checks will run both on the base repo _and_ on the fork (if enabled) on every push to a PR, as well as on every push or merge to the upstream repo. I should however exclude the `gh_pages` branch I suppose?

Reply:
That sounds like what we currently have with Travis if it's enabled on a fork. Is that correct?

Yes, please ignore `gh_pages` :)

Reply:
I believe so. Updated the `on` configuration accordingly.

Reply:
Okay, cool. Thank you!